scripts/mutation-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+DISABLED_MUTATORS='branch/*'
+
+# Only consider the following:
+# * go files in types or keeper packages
+# * ignore test and Protobuf files
+MUTATION_SOURCES=$(find ./x -type f \( -path '*/keeper/*' -or -path '*/types/*' \) \( -name '*.go' -and -not -name '*_test.go' -and -not -name '*pb*' \))
```

Comment:
This is some interestintg file specification syntax lol

Reply:
lol this took me ages to figure out FYI